Code/Framework/AzCore/AzCore/IO/Streamer/Streamer.h

```diff
@@ -174,9 +174,9 @@ namespace AZ::IO
         //! Call to collect statistics from all the components that make up Streamer.
         void CollectStatistics(AZStd::vector<Statistic>& statistics) override;
 
-        //! Tells AZ::IO::Streamer the report the information for the report to the output.
+        //! Tells AZ::IO::Streamer the collect information the selected report type and store it in the provided output.
         FileRequestPtr Report(AZStd::vector<Statistic>& output, IStreamerTypes::ReportType reportType) override;
-        //! Tells AZ::IO::Streamer the report the information for the report to the output.
+        //! Tells AZ::IO::Streamer the collect information the selected report type and store it in the provided output.
```

Comment:
Grammar issue here.
"Tells AZ::IO::Streamer the collect information the selected report type and store it in the provided output."
->
"Tells AZ::IO::Streamer to collect information for the selected report type and store it in the provided output."

Reply:
Will fix before merging this to development.